_blog/installing-docker-and-kubernetes-on-windows/index.md

```diff
@@ -453,6 +451,8 @@ Wordpress is serving traffic from within a container!
 
 > Please note that you won't be able to complete the Wordpress installation because there's no database.
 
+{% include inline-subscribe-cta/index.html %}
```

Comment:
When subscribing, the user is redirected to a confirmation screen with a link to go back to the site. I think the link should return to where the user was and not the top of page. Is this possible?

Reply:
We can customise the page. I don't know if we can customise the link. I'll look into it.

Reply:
I think that if it is not possible, maybe try to remove the confirmation page and simply add a confirmation message on the same page or send a welcome email. I think that if this stays how it is, we will actually lose further reading (time on site) instead of gaining.

Reply:
Does switching to SendInBlue solve the issue?

Reply:
Yes, you can easily add the include used by the newsletter page. However, as it is, we cannot know which page/URL the subscription came from. If you wish, I can prioritise learnk8s/team#44?

Reply:
Just checked and according to the Netlify documentation:

> By default, when visitors complete a form, they will see a generically styled success message with a link back to the form page.

I submitted a test email in the preview and I'm redirected to the following page:

![image](https://user-images.githubusercontent.com/1130499/44298528-ae905700-a2dc-11e8-9941-f9a1f0f56582.png)

The "← Back to our site" link points to the original page `https://deploy-preview-124--learnk8s.netlify.com/blog/scaling-spring-boot-microservices`.

> When subscribing, the user is redirected to a confirmation screen with a link to go back to the site.

Where did you see this?

Reply:
@danielepolencic Yes, this is correct. When a new subscriber clicks on _Back to our site_ they get redirected to the same post BUT to the top of it and not to the section they were reading.